bin/acore-docker-build

```diff
@@ -3,6 +3,10 @@
 docker build -t acbuild -f docker/build/Dockerfile .
 
 docker run \
+    -v /"$(pwd)"/deps:/azerothcore/deps \
+    -v /"$(pwd)"/conf/dist:/azerothcore/conf/dist \
+    -v /"$(pwd)"/src:/azerothcore/src \
+    -v /"$(pwd)"/modules:/azerothcore/modules \
```

Comment:
please also add `-v /"$(pwd)"/.git:/azerothcore/.git \`. if you don't you will end up getting errors like this:
```
fatal: not a git repository (or any of the parent directories): .git
--
    Could not find a proper repository signature (hash) - you may need to pull tags with git fetch -t
    Continuing anyway - note that the versionstring will be set to "unknown 1970-01-01 00:00:00 (Archived)"
```